File: firmware/Event.cpp
#include "Event.h"

Event::Event(int acId, int inten, int tinte, int dura, int pa, int pb)
{
	double dI=inten/100*255;
	double dT=tinte/100*255;
	this->acId=acId;
	this->inten=dI;
	this->tinte=dT;
	this->dura=dura;
	this->pa=pa;
	this->pb=pb;

}
Event::Event(){}

File: GyroPowerball_S/src/at/ac/tuwien/mmi/gyropowerball/Event.java
package at.ac.tuwien.mmi.gyropowerball;

public class Event {

	public int acId;
	public int intensity;
	public int targetIntensity;
	public int duration;
	public int pauseAfter;
	
	public Event(int acId, int intensity, int targetIntensity, int duration, int pauseAfter){
		this.acId=acId;
		this.intensity=intensity;
		this.targetIntensity = targetIntensity;
		this.duration = duration;
		this.pauseAfter = pauseAfter;
	}
	
}
